examples/05-Retrieval-Model.ipynb

```diff
@@ -61,7 +61,8 @@
     "### Learning objectives\n",
```

Comment:
<p>Is there a reason for removing the option <code>no_activation_last_layer</code> ? </p><p> </p><p>And just a quick note about the parameter <code>infer_embedding_sizes</code> :  the parameter is not included in the TwoTowerV2 model as it is set by default in the input block <code>InputBlockV2</code>  </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/900/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>

Reply:
<p>I think it does not really have an affect on the final embedding dim coming out from the towers, so I wont mention about it.</p>